sklearn/utils/tests/test_extmath.py

```diff
@@ -568,6 +567,32 @@ def max_loading_is_positive(u, v):
     assert not v_based
 
 
+def test_randomized_svd_lapack_driver():
+    # Check that different SVD drivers provide consistent results
+
+    # Matrix being compressed
+    n = 123
+    m = 456
+    k = 10
+    rng = np.random.RandomState(0)
+    X = rng.rand(n, m)
+
+    # Number of components
+    k = 10
```

Comment:
How about parametrizing this test on a few combinations of `n`, `m`, `k` and on the random seed?

For instance:
```suggestion
@pytest.mark.parametrize("n", [50, 100, 300])
@pytest.mark.parametrize("m", [50, 100, 300])
@pytest.mark.parametrize("k", [10, 20, 50])
@pytest.mark.parametrize("seed", range(5))
def test_randomized_svd_lapack_driver(n, m, k, seed):
    # Check that different SVD drivers provide consistent results

    # Matrix being compressed
    n = 123
    m = 456
    k = 10
    rng = np.random.RandomState(seed)
    X = rng.rand(n, m)
```

Reply:
I adapted my suggestion which was shadowing the parametrize value.

(It looks like you have not committed all the changes or pushed them.)